Build failure for OpenGL without X11 (GLVND)

Trying to get x86 LibreELEC to [switch from OpenGLES to OpenGL](https://github.com/LibreELEC/LibreELEC.tv/pull/9781). I get this build failure:

> CMake Error at /home/a/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/toolchain/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
>   Could NOT find OpenGl (missing: OPENGL_gl_LIBRARY)
> Call Stack (most recent call first):
>   /home/a/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/toolchain/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
>   FindOpenGl.cmake:31 (find_package_handle_standard_args)
>   CMakeLists.txt:11 (find_package)

https://github.com/smp79/pvr.vdr.vnsi/commit/a3e00de79b4d6f1815bec81decfadf9facfecc86 fixed the build.

--- a/FindOpenGl.cmake
+++ b/FindOpenGl.cmake
@@ -17,7 +17,7 @@ endif()
 if(NOT CORE_SYSTEM_NAME STREQUAL osx)
   find_path(OPENGL_INCLUDE_DIR GL/gl.h
                                PATHS ${PC_OPENGL_gl_INCLUDEDIR})
-  find_library(OPENGL_gl_LIBRARY NAMES GL
+  find_library(OPENGL_gl_LIBRARY NAMES GL OpenGL
                                  PATHS ${PC_OPENGL_gl_LIBDIR})
 else()
   find_library(OPENGL_gl_LIBRARY NAMES OpenGL

